packages/cli/lib/utils.js

```diff
@@ -386,8 +388,96 @@ exports.getRepositoryFileName = function(repositoryName) {
   return `${_.kebabCase(repositoryName)}.repository.ts`;
 };
 
+/**
+ * Returns the serviceName in the directory file format for the service
+ * @param {string} serviceName
+ */
+exports.getServiceFileName = function(serviceName) {
+  return `${_.kebabCase(serviceName)}.service.ts`;
+};
+
+/**
+ *
+ * Returns the connector property for the datasource file
+ * @param {string} datasourcesDir path for sources
+ * @param {string} dataSourceClass class name for the datasoure
+ */
+exports.getDataSourceConnectorName = function(datasourcesDir, dataSourceClass) {
+  if (!dataSourceClass) {
+    return false;
+  }
+  let result;
+  let jsonFileContent;
+
+  let datasourceJSONFile = path.join(
+    datasourcesDir,
+    dataSourceClass.replace('Datasource', '.datasource.json').toLowerCase(),
```

Comment:
Some name mangling is needed to be consistent with datasource.json.

Reply:
I have created a helper function to return the file name correctly and will serve to these two functions for now in utils.js . Thank you for finding that bug @raymondfeng